monai/run_inference_single_image.py

```diff
@@ -72,6 +70,8 @@ def get_parser():
                         ' Default: 64x192x-1')
     parser.add_argument('--device', default="gpu", type=str, choices=["gpu", "cpu"],
                         help='Device to run inference on. Default: cpu')
+    parser.add_argument('--use-tta', action='store_true', 
+                        help='Use test-time augmentation (TTA) i.e mirroring on all axes. Default: False')
```

Comment:
nitpick suggestion just to be consistent with [line 234](https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/136aff1b84b61be74ad57db3c6dc73c6ffef6685/monai/run_inference_single_image.py#L234)

```suggestion
    parser.add_argument('--use-tta', action='store_true', 
                        help='Use test-time augmentation (TTA), i.e. mirroring across all 3 axes. Default: False')
```